src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -140,6 +140,10 @@ const CreateOrEditQuizContainer = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [id])
 
+  useEffect(() => {
+    isOpen && setIsCreationOpen(false)
```

Comment:
do you need this state, maybe you can use isOpen instead of isCreationOpen variable?

Reply:
yeah, you are right